src/main/resources/subject-contact-details/translation.json

```diff
@@ -1,56 +1,56 @@
-        },
-        "cy": {
-            "continue": "Parhau",
-            "serviceName": "Apelio i’r Tribiwnlys Haen Gyntaf",
-            "title": "Rhowch fanylion cyswllt",
-            "line1": "Rhowch y manylion cyswllt sydd fwyaf addas ar gyfer yr achos hwn.",
-            "emailAddressLabel": "Cyfeiriad e-bost",
-            "emailAddressHint": "",
-            "emailAddress": "Nodwch gyfeiriad e-bost",
-            "contactNumberLabel": "Rhif cyswllt",
-            "contactNumberHint": "",
-            "agreeContactLabel": "Rwy’n cytuno i’r llys gysylltu â mi am yr achos hwn gan ddefnyddio’r manylion rwyf wedi’u darparu"
-        }
+  "translations": {
+    "en": {
+      "continue": "Continue",
+      "serviceName": "Appeal to the First-tier Tribunal",
+      "title": "Enter contact information",
+      "line1": "Please enter your contact details to use for this case.",
+      "emailAddressLabel": "Email address",
+      "emailAddressHint": "Enter an email address in the correct format, like name@example.com",
+      "emailAddress": "Insert email address",
+      "contactNumberLabel": "Contact number",
+      "contactNumberHint": "Enter a telephone number in the correct format, like 01632 960 001 or 07700 900 982",
+      "agreeContactLabel": "I agree to be contacted about this case using the details I have provided"
     },
-    "errors":{
-        "en": {
-            "subjectEmailAddress": {
-                "required": "Please enter a valid email address",
-                "invalid": "Every email address consists of 3 elements: local-part, @ symbol (pronounced as “at”), and a domain name"
-            },
-            "subjectContactNumber": {
-                "required": "Please enter a valid contact number",
-                "invalid": "Please enter a valid contact number"
-            },
-            "subjectAgreeContact": {
-                "required" : "Please agree to be contacted using the details provided to continue"
-            }
-        },
-        "cy": {
-            "subjectEmailAddress": {
-                "required": "Nodwch gyfeiriad e-bost dilys",
-                "invalid": "Mae pob cyfeiriad e-bost yn cynnwys tair elfen: rhan lleol, symbol @ (a ynganir fel “at”), ac enw parth"
-            },
-            "subjectContactNumber": {
-                "required": "Nodwch rif cyswllt dilys",
-                "invalid": "Nodwch rif cyswllt dilys"
-            },
-            "subjectAgreeContact": {
-                "required" : "Dylech gytuno i’r llys gysylltu â chi gan ddefnyddio’r manylion a ddarparwyd cyn y gallwch barhau"
-            }
-        }
+    "cy": {
+      "continue": "Parhau",
+      "serviceName": "Apelio i’r Tribiwnlys Haen Gyntaf",
+      "title": "Rhowch fanylion cyswllt",
+      "line1": "Rhowch y manylion cyswllt sydd fwyaf addas ar gyfer yr achos hwn.",
+      "emailAddressLabel": "Cyfeiriad e-bost",
+      "emailAddressHint": "",
```

Comment:
Don't we want to give hints in welsh?

Reply:
Welsh translation is out of the scope of this ticket (as declared in the description of this PR)